src/sentry/api/endpoints/group_hashes.py

```diff
@@ -32,15 +41,30 @@ def get(self, request, group):
         :auth: required
         """
 
-        queryset = GroupHash.objects.filter(
-            group=group.id,
+        aggregations = [
+            ('argMax(event_id, timestamp)', None, 'event_id')
+        ]
+
+        filter_keys = {
+            'project_id': [group.project_id],
+            'group_id': [group.id]
+        }
+
+        data_fn = partial(
+            lambda *args, **kwargs: raw_query(*args, **kwargs)['data'],
+            start=datetime.utcfromtimestamp(0),  # will be clamped to project retention
+            end=datetime.utcnow(),  # will be clamped to project retention
+            aggregations=aggregations,
+            filter_keys=filter_keys,
+            selected_columns=['primary_hash'],
+            groupby=['primary_hash'],
+            referrer='api.group-hashes',
```

Comment:
Doesn't this need an `ORDER BY` clause to ensure pagination is deterministic? I don't think one is added implicitly, and even if it is, it seems like it'd be clearer for it to be explicitly added here.